app/components/LiveDock.vue

```diff
@@ -38,14 +38,13 @@
         </div>
         <div class="live-dock-viewer-count">
           <i
-            class="fa fa-eye live-dock-viewer-count-toggle label--icon"
             :class="{
-              'fa-eye': !hideViewerCount,
-              'fa-eye-slash': hideViewerCount
+              'icon-view': !hideViewerCount,
+              'icon-hide': hideViewerCount
             }"
             @click="toggleViewerCount"/>
-          <i class="fa fa-user label--icon" />
-          <span class="weight--medium">{{ viewerCount }}</span> {{ $t('viewers')}}
+          <!-- <i class="icon-team label--icon" /> -->
```

Comment:
Are we removing this icon intentionally?  It looks like you updated the icon class but commented it out, so I wanted to make sure this was intended.